File: tests/ClaudeToolbar.Core.Tests/Credentials/CredentialsPathsTests.cs
using ClaudeToolbar.Core.Credentials;

namespace ClaudeToolbar.Core.Tests.Credentials;

public class CredentialsPathsTests
{
    [Fact]
    public void DefaultsToUserProfileClaudeDir()
    {
        var path = CredentialsPaths.Resolve(null, @"C:\Users\me");
        Assert.Equal(Path.Combine(@"C:\Users\me", ".claude", ".credentials.json"), path);
    }

    [Fact]
    public void EnvOverrideWins()
    {
        var path = CredentialsPaths.Resolve(@"D:\cfg", @"C:\Users\me");
        Assert.Equal(Path.Combine(@"D:\cfg", ".credentials.json"), path);
    }

    [Fact]
    public void BlankOverrideIsIgnored()
    {
        var path = CredentialsPaths.Resolve("   ", @"C:\Users\me");
        Assert.Equal(Path.Combine(@"C:\Users\me", ".claude", ".credentials.json"), path);
    }
}

File: mac/Tests/ClaudeToolbarCoreTests/Credentials/CredentialsPathsTests.swift
import XCTest
@testable import ClaudeToolbarCore

final class CredentialsPathsTests: XCTestCase {
    func testDefaultUnderHome() {
        let path = CredentialsPaths.resolve(claudeConfigDir: nil, homeDirectory: "/Users/sam")
        XCTAssertEqual(path, "/Users/sam/.claude/.credentials.json")
    }

    func testBlankOverrideIsIgnored() {
        XCTAssertEqual(CredentialsPaths.resolve(claudeConfigDir: "   ", homeDirectory: "/Users/sam"), "/Users/sam/.claude/.credentials.json")
    }

    func testOverrideWins() {
        XCTAssertEqual(CredentialsPaths.resolve(claudeConfigDir: " /tmp/cfg ", homeDirectory: "/Users/sam"), "/tmp/cfg/.credentials.json")
    }
}
